muffnn/mlp/mlp_classifier.py

```diff
@@ -84,7 +84,7 @@ class MLPClassifier(MLPBaseEstimator, ClassifierMixin):
     def __init__(self, hidden_units=(256,), batch_size=64, n_epochs=5,
                  keep_prob=1.0, activation=nn.relu, init_scale=0.1,
                  random_state=None, solver=tf.train.AdamOptimizer,
-                 solver_kwargs=None):
+                 solver_kwargs=None, transform_layer_index=0):
```

Comment:
I think this should default to `None` if you want to have the default be the last layer